### DISCUSSION TITLE
Content validation

### DISCUSSION THREAD

**Discussion by ciwchris** [SimonCropp/MarkdownSnippets#559] [Q&A] [answered]
When I started using this tool I was surprised to find it was also enforcing content validation.

> Invalid word detected: 'we'

Although a nice feature it doesn't seem directly related to the purpose of this tool. I think it would be helpful to document this feature and/or create a config option to enable or disable the feature. Thoughts?

**Comment by SimonCropp** [comment, marked as answer]
do u have ValidateContent enabled?

**Reply by ciwchris** [reply]
Agh. Thank you. I pasted in the config and my eyes did not catch it.
